src/content/configuration/node.md

```diff
@@ -46,38 +35,40 @@ module.exports = {
 };
 ```
 
-Since webpack 3.0.0, the `node` option may be set to `false` to completely turn off the `NodeStuffPlugin` and `NodeSourcePlugin` plugins.
+Since webpack 3.0.0, the `node` option may be set to `false` to completely turn off the `NodeStuffPlugin` plugin.
 
 ## `node.global`
 
-`boolean = true`
+`boolean`
 
-Defaults to `false` for [targets](/configuration/target/) `node`, `async-node` and `electron-main`.
+T> If you are using a module which needs global variables in it, use `ProvidePlugin` instead of `global`.
 
-See [the source](https://nodejs.org/api/globals.html) for the exact behavior of this object.
+See [the Node.js documentation](https://nodejs.org/api/globals.html#globals_global) for the exact behavior of this object.
 
+Options:
 
-## `node.__filename`
+- `true`: Provide a polyfill.
+- `false`: Provide nothing. Code that expects this object may crash with a `ReferenceError`.
 
-`boolean` `string = mock`
+## `node.__filename`
 
-Defaults to `false` for [targets](/configuration/target/) `node`, `async-node` and `electron-main`.
+`boolean` `string: 'mock' | 'eval-only'`
 
 Options:
 
 - `true`: The filename of the __input__ file relative to the [`context` option](/configuration/entry-context/#context).
-- `false`: The regular Node.js `__filename` behavior. The filename of the __output__ file when run in a Node.js environment.
-- `'mock'`: The fixed value `'index.js'`.
+- `false`: Webpack won't touch your `__filename` code, which means you have the regular Node.js `__filename` behavior. The filename of the __output__ file when run in a Node.js environment.
```

Comment:
```suggestion
- `false`: webpack won't touch your `__filename` code, which means you have the regular Node.js `__filename` behavior. The filename of the __output__ file when run in a Node.js environment.
```

Reply:
I think Capital `W` is allowed now when `webpack` occurs in the beginning https://webpack.js.org/blog/2020-10-10-webpack-5-release/#other-minor-changes :)

![image](https://user-images.githubusercontent.com/1091472/96576441-c6bcf000-1304-11eb-93f4-5d66a47a5011.png)